js/tests/unit/util/index.spec.js

```diff
@@ -660,6 +661,29 @@ describe('Util', () => {
     })
   })
 
+  describe('removeElement', () => {
+    it('should removeElement if arg is element and return true', () => {
+      fixtureEl.innerHTML = '<div></div>'
+      const div = fixtureEl.querySelector('div')
+
+      expect(removeElement(div)).toBe(true)
+      expect(fixtureEl.querySelectorAll('div').length).toBe(0)
+    })
+
+    it('should return false if no argument given', () => {
+      expect(removeElement()).toBe(false)
+      expect(removeElement({})).toBe(false)
+    })
+
+    it('should return false if argument is element but it is removed mefore execution', () => {
```

Comment:
@alpadev Lets say we use `element.remove()`

What result should we get in this case?

Reply:
Why do you want to return something? Judging from how we use it, looks like we don't use the return value at all.

Reply:
And in case you want to return something you could do it like
```js
const removeElement = (element) => {
  try {
    element.remove()
  } catch {
    return false
  }

  return true
}
```

Reply:
Used to return results, I was thinking of this implementation.
It is not really important to renturn something

Reply:
As you wish. If you feel you want to return something that's ok I guess. I'm just not sure how to use that value or if there is any benefit to it.

I think it's ok to rely on the browser to remove an element. So to assume if you called remove with some valid element it should be removed. The old implementation had the problem checking explicitly for the `parentNode` that might be absent if the element was removed already. With `.remove()` the browser is handling this case automatically.

From another perspective it's not some public function that is used here. That said, I'm not sure if we can assume that element will always be a valid element, that would make any sort of parameter validation unnecessary.

I was thinking about to use `el.remove()` directly without wrapping it. From one side it should do the job, from another perspective we may save some bits by using a wrapper function. :)

I trust in you to make the best decision. ❤️

Reply:
So you think something like this:
```js
const removeElement = arg => {
  const element = getElement(arg)
  if (element) {
    element.remove()
  }
}

```

Reply:
I was thinking to simply use `element.remove()` because we know that we use it for elements only, but do as you like.

Reply:
didn't caught your thought :/

Reply:
Agree with @alpadev :100:
`element.remove()` should be used directly IMO, but in some cases, the `element` may be `null`.

So checking for the truthy value is better
```js
if (element) {
  element.remove()
}
```